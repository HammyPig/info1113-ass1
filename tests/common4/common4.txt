one book intersection, with complex transactions of books