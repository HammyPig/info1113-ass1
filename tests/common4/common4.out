user: Success.

user: Success.

user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: The Hobbit (J.R.R. Tolkien)

user: Ending Library process.
